wpilibc/src/main/native/cpp/shuffleboard/ShuffleboardValue.cpp

```diff
@@ -0,0 +1,18 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "frc/shuffleboard/ShuffleboardValue.h"
+
+using namespace frc;
+
+ShuffleboardValue::ShuffleboardValue(wpi::StringRef title) { m_title = title; }
+
+const std::string& ShuffleboardValue::GetTitle() const { return m_title; }
```

Comment:
Either return a `StringRef` or a `std::string` by value instead of returning a `const std::string&`.  If the title can never change after construction it's safe to use a StringRef, otherwise use a `std::string`.